api/test/pact/pact-tests/pact-config.ts

```diff
@@ -0,0 +1,16 @@
+// import nodeConfig from 'config';
+//
+// interface Config {
+//   pactRdProfessionalApi:string;
+//   pactFeeAndPayApi:string;
+// }
+//
+// const config: Config = {
+//   pactRdProfessionalApi: nodeConfig.get<string>('services.rdProfessionalApi'),
+//   // tslint:disable-next-line:object-literal-sort-keys
+//   pactFeeAndPayApi: nodeConfig.get<string>('services.feeAndPayApi'),
+// };
+//
```

Comment:
Whats the use of this file? if its commented out

Reply:
this has been removed now Satya